hassio/src/dashboard/hassio-addons.ts

```diff
@@ -70,35 +86,18 @@ class HassioAddons extends LitElement {
     ];
   }
 
-  private _computeIcon(addon: HassioAddonInfo): string {
-    return addon.installed !== addon.version
-      ? "hassio:arrow-up-bold-circle"
-      : "hassio:puzzle";
-  }
-
-  private _computeIconTitle(addon: HassioAddonInfo): string {
-    if (addon.installed !== addon.version) {
-      return "New version available";
-    }
-    return addon.state === "started"
-      ? "Add-on is running"
-      : "Add-on is stopped";
-  }
-
-  private _computeIconClass(addon: HassioAddonInfo): string {
-    if (addon.installed !== addon.version) {
-      return "update";
-    }
-    return addon.state === "started" ? "running" : "";
-  }
-
   private _addonTapped(ev: any): void {
     navigate(this, `/hassio/addon/${ev.currentTarget.addon.slug}`);
   }
 
   private _openStore(): void {
     navigate(this, "/hassio/store");
   }
+
+  private get _computeHA105plus(): boolean {
```

Comment:
Personally not a fan of getters when doing computation. It's only 1 call here so it's ok.